mergebot-config.json

```diff
@@ -52,8 +53,13 @@
                     "teamcity/dcos/test/aws/onprem/static/group2",
                     "teamcity/dcos/test/aws/onprem/static/group3",
                     "teamcity/dcos/test/aws/onprem/static/group4",
-                    "teamcity/dcos/test/test-e2e/group1"],
```

Comment:
I've changed the formatting to make this (and future) changes a little simpler.

Reply:
Thanks!  sgtm.